src/os/interaction/modifyinteraction.js

```diff
@@ -225,6 +229,34 @@ class Modify extends OLModify {
     this.removeControls();
   }
 
+  /**
+   * @inheritDoc
+   */
+  setActive(value) {
+    const changed = value !== this.getActive();
+
+    if (changed && this.clone_) {
+      if (!value) {
```

Comment:
This logic to manage the clone listener and map feature are also in the constructor and dispose function. Do we still want them there?

Reply:
I don't think we're doing it anywhere currently, but if something wanted to deactivate the interaction without disposing the interaction, I think we'd still want this logic to remove the feature from the map.